src/views/Main/Catalog/Components/EntitySetList.js

```diff
@@ -48,9 +149,54 @@ export class EntitySetList extends React.Component {
       />);
     });
     return (
-      <div>
+      <div className={styles.edmContainer}>
```

Comment:
this is pretty huge. are there pieces of this HTML that can be reused in other places of the app? if so, it would be good to split that out into more modular components.

this is fine for now, but I recommend thinking about how you might refactor this to be more generic so that you can reuse as much as possible.